### PR TITLE
Prettify PR's via GH-actions

### LINKED ISSUE #689: Add prettier to format code
**Describe the solution you'd like**
Add prettier and a prettier config to format code because every time someone edits a file, it uses their formatting settings. This will break diffs.

### PR DESCRIPTION
**Pull request by yashash1511**
Closes #689 

### Type of change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by andrewstech — APPROVED** [review]
Pretty

**Comment by andrewstech** [comment]
Pretty Pr will probably need a filter not to trigger on win11bots changes

**Comment by yashash1511** [comment]
> Pretty Pr will probably need a filter not to trigger on win11bots changes

sorry i don't exactly understand what does this mean and when will it apply.

**Comment by andrewstech** [comment]
> > Pretty Pr will probably need a filter not to trigger on win11bots changes
> 
> sorry i don't exactly understand what does this mean and when will it apply.

So when the action pushed the changes it trigged its self.

**Comment by yashash1511** [comment]
> this

ohh i see

**Comment by andrewstech** [comment]
> > this
> 
> ohh i see any idea how can we implement that?

yes

**Comment by yashash1511** [comment]
yes we can add skip ci to the commit msg that shall do it

File: .github/workflows/Prettier-PR.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ permissions: write-all
 
 jobs:
   prettier:
+    if: "!contains(github.actor, 'win11bot')" #skips if win11bot trigged pr
```
Comment (yashash1511) [inline .github/workflows/Prettier-PR.yaml:11]: nic

Comment (andrewstech) [reply .github/workflows/Prettier-PR.yaml:11]: Got to test it

Comment (andrewstech) [reply .github/workflows/Prettier-PR.yaml:11]: Ok it works

**Comment by yashash1511** [comment]
gg

**Comment by blueedgetechno** [comment]
God bless you,

**Pull request merged**